panel/widgets/base.py

```diff
@@ -96,13 +97,29 @@ def from_param(cls, parameter: param.Parameter, **params) -> Self:
         )
         return layout[0]
 
+    @property
+    def _linked_properties(self) -> Tuple[str]:
```

Comment:
The type should be `Tuple[str, ...]` unless the expectation is for a single value tuple.